app/src/main/java/com/mashup/lastgarden/ui/scent/ScentFragment.kt

```diff
@@ -175,12 +178,16 @@ class ScentFragment : BaseViewModelFragment(), ScentViewPagerAdapter.OnClickList
         }
     }
 
-    override fun onCommentClick(scentId: Int) {
-        ScentCommentBottomSheetFragment().show(requireActivity().supportFragmentManager, "")
+    override fun onCommentClick(storyId: Int) {
+        ScentCommentBottomSheetFragment().apply {
+            val bundle = Bundle()
+            bundle.putInt("storyId", storyId)
+            arguments = bundle
+        }.show(requireActivity().supportFragmentManager, "")
```

Comment:
```suggestion
        }.show(requireActivity().supportFragmentManager, null)
```
구분하지 않을거면 null이 더 깔끔해보이네요